### ISSUE TITLE
Context menu commands not showing when right-click on table range

### ISSUE THREAD

**Issue by Nikodem98** [OfficeDev/office-js#4692]
Context menu commands not showing when right-click on table range on macOS desktop and windows desktop Excel but works perfectly fine on web Excel.

Any way to fix this?

![image](https://github.com/user-attachments/assets/a552aef0-1324-4c68-908e-a57bdafc7720)
![image](https://github.com/user-attachments/assets/3feb7af9-d2da-4a7b-bd76-e768215482dd)

**Comment by AlexJerabek** [comment]
Hi @Nikodem98,

Thanks for reporting this. @EsterBergen, could you please investigate?

**Comment by EsterBergen** [comment]
@Nikodem98 - can you provide more details about this issue? Are the context menus coming from a specific addin that was enabled for your sheet?

**Comment by Nikodem98** [comment]
Yes, I created a addin that has some commands in right click menu. Everything works when I select a single cell or a range that is not a table. When I right click on selected table range commands from context menu are gone.
It happens only on excel Mac/windows desktop application, on web excel works perception fine

**Comment by EsterBergen** [comment]
Thanks for clarifying! We'll investigate and get back to you soon,

**Comment by EsterBergen** [comment]
@Nikodem98 - and another clarifying question. When you say Table Range, is it purely a group of selected cells or is this range defined as a table in excel?

**Comment by Nikodem98** [comment]
A range defined as a table. With group of cell with values it works fine( first screen shoot), but when I select a cells range defined as a table my context menu commands are gone (second screenshot)
![image](https://github.com/user-attachments/assets/04c6ce06-696f-4012-a42c-66966ab20ebb)
![image](https://github.com/user-attachments/assets/774cc597-c05d-48ea-9986-6f9276b9fb30)

as you can see there is a table ribbon at te top 
And the command are gone

**Comment by EsterBergen** [comment]
@Nikodem98 - Thanks again for reporting this feature gap. We've created an item in the backlog to bring win32/Mac to parity with web. (9177817).  We will post a message here once we get a better sense on when we will be able to make this fix. Thanks!
